src/jens/repos.py

```diff
@@ -299,6 +299,8 @@ def _expand_clones(partition, name, inventory, inventory_lock, new_refs,
             # mid-flight.
             git.fetch(clone_path)
             git.reset(clone_path, "origin/%s" % refname, hard=True)
+            logging.info("Updated ref '%s' (%s)" % (clone_path,
```

Comment:
Yes, actually it's more efficient. However, it's done like this so the way strings are formatted is consistent across the whole application.